public/app/core/table_model.ts

```diff
@@ -99,11 +99,14 @@ export function mergeTablesIntoModel(dst?: TableModel, ...tables: TableModel[]):
     return model;
   }
 
+  // Filter out any tables that are not of TableData format
+  const tableDataTables = tables.filter(table => !!table.columns);
```

Comment:
Does this mean that this code can go away?
https://github.com/grafana/grafana/blob/46a4118461a02f7f7146696ee00715975eddcbca/public/app/features/explore/utils/ResultProcessor.ts#L39

Reply:
don't think so that is something else, without that Prometheus results will show time series in graph and table (where we only want table (instant) query result in table).